cfme/utils/ssh.py

```diff
@@ -262,14 +262,13 @@ def get_transport(self, *args, **kwargs):
             self.connect()
         return super(SSHClient, self).get_transport(*args, **kwargs)
 
-    def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, ensure_host=False,
+    def run_command(self, command, timeout=RUNCMD_TIMEOUT, ensure_host=False,
                     ensure_user=False, container=None):
         """Run a command over SSH.
 
         Args:
             command: The command. Supports taking dicts as version picking.
             timeout: Timeout after which the command execution fails.
-            reraise: Does not muffle the paramiko exceptions in the log.
```

Comment:
Q: can you please explain why we are removing `reraise`?

Reply:
sorry for this inconvenience but please check https://github.com/ManageIQ/integration_tests/pull/8749. I had to create separate PR for this commit and the reason is written in the PR desc.